pkg/ccl/crosscluster/logical/lww_kv_processor.go

```diff
@@ -95,10 +95,6 @@ var originID1Options = &kvpb.WriteOptions{OriginID: 1}
 func (p *kvRowProcessor) ProcessRow(
 	ctx context.Context, txn isql.Txn, keyValue roachpb.KeyValue, prevValue roachpb.Value,
 ) (batchStats, error) {
-	if err := p.injectFailure(); err != nil {
```

Comment:
nit: why was this moved a couple lines down?

Reply:
I moved it after `p.lastRow = row` bc we want to be able to read the decoded row from the processor for the dlq test

Reply:
ahhhh. ty for explaining for the dull